### PR TITLE
Daily Active Users, script and docs

### PR DESCRIPTION
**Pull request by amynic**
### Issue: #500 

 - [ ] Tick here to confirm that you have run RunCodeFormatter.ps1

### Description:
Created DAU.usql script which integrates with message formatters in Nether.Analytics. Script computes unqiue daily active users from Data Lake Store and outputs a .csv FILE

### Test:
Create some session start data with at least two users and have each user with different sessions
Build the DAU.usql script
submit the script against ADLA
check output in nether/dau/results folder as .csv

### CODE REVIEW

File: documentation/analytics/recipes/daily-active-users.md [hunk 1]
```diff
@@ -1,2 +1,162 @@
...
+* Or on a __monthly__ basis (28-31 Dates, 28-31 Counts)
+
+
+| Query that should be run           | By what service                           |
+|------------------------------------|-------------------------------------------|
+| [DAU.usql](../../../src/Nether.Analytics.DataLake.Jobs/DAU.usql) | Azure Data Lake Analytics |
+
+#### 3.1 Setup and Schedule Query1
+
+* Open the Nether.Analytics Solution
```
Comment (krist00fer) [inline documentation/analytics/recipes/daily-active-users.md]: Scip this section for now. We are just about to have that code ready, so even though I do like the ilustrations and the way you described executing the job, we are not going to need it. It's all going to be code :-)

File: documentation/analytics/recipes/daily-active-users.md [hunk 2]
```diff
@@ -1,2 +1,162 @@
 # Daily Active Users, DAU
 
+__Daily Active Users (DAUs)__ is the number of unique users that start at least one session in your app on any given day. DAU is a high level metric about the usage of your game over time. 
+
+## End Result
+
+ A DAU count is a snapshot of time, for example on 17/05/2017 a game has 10,000 DAU. [date, count]
```
Comment (krist00fer) [inline documentation/analytics/recipes/daily-active-users.md]: Awesome, could you please change the time format to the format we use everywhere else. YYYY-MM-DD

File: documentation/analytics/recipes/daily-active-users.md [hunk 3]
```diff
@@ -1,2 +1,162 @@
...
+| Date         | Count          |
+| -------------|:--------------:|
+| 17/05/2017   | 10,000         |
+| 18/05/2017   | 11,000         |
+| 19/05/2017   | 10,500         |
+| 20/05/2017   | 11,000         |
+| 21/05/2017   | 12,000         |
+| 22/05/2017   | 13,000         |
+
+![DAU over Time Graph](../../images/analytics/DAUReciepe/dauovertime.jpg)
```
Comment (krist00fer) [inline documentation/analytics/recipes/daily-active-users.md]: This graph is perfect 👍

File: documentation/analytics/recipes/daily-active-users.md (hunk 2 above)
Comment (amynic) [inline documentation/analytics/recipes/daily-active-users.md]: updated to '2017-05-17T00:00:00.0000000-07:00' (yyyy-mm-dd)

File: documentation/analytics/recipes/daily-active-users.md (hunk 3 above)
Comment (amynic) [inline documentation/analytics/recipes/daily-active-users.md]: Great news! :)

File: documentation/analytics/recipes/daily-active-users.md (hunk 1 above)
Comment (amynic) [inline documentation/analytics/recipes/daily-active-users.md]: removed and added <under development to automate>

File: src/Nether.Analytics.DataLake.Jobs/DAU.usql [hunk 4]
```diff
@@ -0,0 +1,35 @@
+﻿//PURPOSE: A USQL job to compute the Daily Active Users (DAUs) of the game 
+// RECOMMENDED: run transactional eventdata daily
+
+
+//Declare input/output destinations as variables
```
Comment (krist00fer) [inline src/Nether.Analytics.DataLake.Jobs/DAU.usql]: Parameters should be DECLARE EXTERNAL in order for us to replace them in runtime

**Comment by krist00fer** [comment]
I've re-based and fixed a problem with variables not declared as "DECLARE EXTERNAL". Ready to merge now

**Pull request merged**